<ui version="4.0" >
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="SearchFormWidget" native="1" name="form" />
   </item>
   <item>
    <widget class="Screen" native="1" name="screen" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushFind" >
         <property name="text" >
          <string>&amp;Find</string>
         </property>
         <property name="icon" >
          <iconset resource="common.qrc" >
           <normaloff>:/images/images/find.png</normaloff>:/images/images/find.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushAccept" >
         <property name="text" >
          <string>&amp;Accept</string>
         </property>
         <property name="icon" >
          <iconset resource="common.qrc" >
           <normaloff>:/images/images/ok.png</normaloff>:/images/images/ok.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCancel" >
         <property name="text" >
          <string>&amp;Cancel</string>
         </property>
         <property name="icon" >
          <iconset resource="common.qrc" >
           <normaloff>:/images/images/cancel.png</normaloff>:/images/images/cancel.png</iconset>
         </property>
         <property name="shortcut" >
          <string>Esc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchFormWidget</class>
   <extends>QWidget</extends>
   <header>Search</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Screen</class>
   <extends>QWidget</extends>
   <header>Screen</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="common.qrc" />
 </resources>
 <connections/>
</ui>
